Setup/MakeMonoArchive.cmd

```diff
@@ -1,43 +1,44 @@
 @echo off
```

Comment:
I would also rename this to something like `MakePortableArchive.cmd`

Reply:
Suggest this for a separate issue/PR

Reply:
ok, nw